public/js/AdminCtrl.js

```diff
@@ -45,8 +45,8 @@ polr.controller('AdminCtrl', function($scope, $compile) {
                 "ajax": BASE_API_PATH + 'admin/get_admin_links',
 
                 "columns": [
-                    {className: 'wrap-text', data: 'short_url', name: 'short_url'},
-                    {className: 'wrap-text', data: 'long_url', name: 'long_url'},
+                    {data: 'short_url', name: 'short_url'},
+                    {data: 'long_url', name: 'long_url', width: '30%'},
```

Comment:
I tested both the previous tables (pre this change) and this pull, both of them seem to function poorly on mobile. 
[Screenshot from polr demo admin](http://puu.sh/sVaj1/401023a6bf.png)